### ISSUE TITLE
Newest Pylance/Pyright is reporting incorrect errors that pyright CLI does not

### ISSUE THREAD

**Issue by VictorPeralta** [microsoft/pyright#3479]
**Describe the bug**

**To Reproduce**
On Pylance v2022.5.2, with [django-types](https://github.com/sbdchd/django-types), VS Code reports an incorrect error.

Too few type arguments provided for "ForeignKey"; expected 2 but received 1Pylance[reportGeneralTypeIssues](https://github.com/microsoft/pylance-release/blob/main/DIAGNOSTIC_SEVERITY_RULES.md#diagnostic-severity-rules) over `Source` in the following django model code snippet.

```
    source = models.ForeignKey[Source](
        Source,
        on_delete=models.CASCADE,
        related_name="feedback",
    )
``` 

**Expected behavior**
No error should be thrown. After installing pyright and running it on this codebase, only correct errors are thrown.

**VS Code extension or command-line**
Pylance v2022.5.2
VS Code v1.67.1
OS: Ubuntu 20.04.4 LTS
**Additional context**
Downgrading to Pylance v2022.4.3 makes the incorrect errors go away

Found this issue opened previously which is similar to my issue
https://github.com/microsoft/pyright/issues/2631

**Comment by erictraut** [comment]
Could you please provide a minimal self-contained code sample? The code snippet above references symbols that are not built in to Python, and I don't know which libraries they are imported from. Please also include the versions of the library or libraries that you are importing.

**Comment by VictorPeralta** [comment]
Thanks for the quick response. I'll provide this information tomorrow!

**Comment by erictraut** [comment]
This will be caused by differences in configuration between Pylance and the CLI version of Pyright. If you're not able to diagnose the problem, please provide additional details and I can help.

I'm going to close the issue because it's almost definitely not a bug.

**Comment by B4rtware** [comment]
Since I landed here after I also had this problem I'll write here how I fixed it. In my case it was due to the automatically supplied stub files from pylance. Pylance has extra stub files for django which don't get along with `django-types`. Deleting or renaming the folder `~\.vscode-insiders\extensions\ms-python.vscode-pylance-2022.7.44\dist\bundled\stubs\django` (or `.vscode` and another version) solved the problem.
